testing.rst

```diff
@@ -505,7 +505,7 @@ afterwards with the ``followRedirect()`` method::
     $crawler = $client->followRedirect();
 
 If you want the client to automatically follow all redirects, you can
-force him with the ``followRedirects()`` method::
+force him by calling the ``followRedirects()`` method before performing the request::
```

Comment:
Love your addition! If you have the time, please make one more minor change in this change: Change "him" to "them". We want to be as gender-neutral as possible in the docs :)